### ISSUE TITLE
After update to 1.7.8.0, can't edit product Summary/Description

### ISSUE THREAD

**Issue by xtofl** [PrestaShop/PrestaShop#26543]
#### Describe the bug

I used the 1-click update to go from 1.7.4 to 1.7.8.0.

After update, I can't edit product details: no edit field is present underneath the "Summary" or "Description" titles.

#### Expected behavior

The edit fields are present.

#### Steps to Reproduce

1. Go to 'Catalog/Products'
2. Click on 'New product'
3. See error: you can edit the single-line entries, but not the 'big ones' like Summary and Description.

**Screenshots**

![1415497102_2021-10-2114_33_25-Window png 580cba1e932e012f0ccc62db0a9c0dc6](https://user-images.githubusercontent.com/441092/140646611-18ba4c90-7c2e-43c4-bf86-2d3fc38805c7.png)

#### Additional information

* Also mentioned on forum: https://www.prestashop.com/forums/topic/1055811-unable-to-edit-page-content-product-titles-product-summary-etc-after-upgrading-from-1760-to-1780/.  I did find a workaround by changing the tiny_mce theme to 'mobile'.

* PrestaShop version: 1.7.8.0
* PHP version: 7.4

**Comment by hibatallahAouadni** [comment]
Hello @xtofl 

I upgraded a 1.7.4.1 shop to 1.7.8.0 and I didn't encounter any issue, see the attached screen record below:

https://watch.screencastify.com/v/Zpc9wj62SfZXai9ZkDTB

Please check and feedback.
Thanks!

**Comment by xtofl** [comment]
@hibatallahAouadni thank you for your time.  I noticed that the PHP version in the screen cast is 7.1.

I remember while trying-and-erroring to solve the problem, before I tried the upgrade,  choosing PHP 7.3, also solved it.  I'll try reverting to 7.3 (my hosting service doesn't allow going back further) and come back.

**Comment by gs2k84** [comment]
Hello,

did you solve the problem?
I had exactly the same after updating from 1.7.7.7 to 1.7.8.0

Hopefully in 1.7.8.1 it won't be there anymore.

**Comment by florine2623** [comment]
Hello @xtofl ,

It is better to use PHP 7.1 when you upgrade from PS 1.7.4.0 (this version is only compatible until PHP 7.1.)
Once the upgrade is done, you can update your PHP version to 7.4.
Here, you will find more info about the system requirements : https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/

Thanks!

**Comment by przemokon** [comment]
Hello,

Today i upgraded PS from 1.7.7.4 to 1.7.8.1 on my dev shop and have same problem.
I have errors in console on product or CMS page (both loads tinymce plugin).
`Uncaught Error: Could not find control by type: button`

Tested on both 7.3 and 7.4 PHP versions.

Edit
Tested with [tinymce 4.9.8](https://raw.githubusercontent.com/PrestaShop/PrestaShop/8ee406a0c5ae18a69d5f0d36de5a00e351d2b80f/js/tiny_mce/tinymce.min.js) with no luck, only v 4.0.16 works properly (version before 4.9.8)

**Comment by khouloudbelguith** [comment]
Hi @xtofl, @przemokon, @gs2k84,

Could you please try to configure your smarty by disabling the cache, set the option "Template compilation" to Force compilation, and clear the cache.
Also, try to open your shop in an incognito browser.
You need to force refresh the page, by pressing CTRL+F5 on Windows / Linux, Cmd+R on Mac OS while you are on this page.

Thanks to check and feedback.

**Comment by gs2k84** [comment]
I did as you wrote and it did not work.

The description still when editing the product appears for half a second and disappears.

However, copying tiny_mce from version 1.7.7.7 helped
Everything works now.



Interestingly, in a clean installation of prestashop 1.7.8.0 on the same server, everything works fine.

**Comment by przemokon** [comment]
Hi @khouloudbelguith, in my case that was helpful.
After logging into incognito mode tinymce starts working properly.

Thanks

**Comment by khouloudbelguith** [comment]
> I did as you wrote and it did not work.
> 
> The description still when editing the product appears for half a second and disappears.
> 
> However, copying tiny_mce from version 1.7.7.7 helped Everything works now.
> 
> Interestingly, in a clean installation of prestashop 1.7.8.0 on the same server, everything works fine.



@gs2k84, I did not manage to reproduce the issue after an upgrade from PS1778 to PS1781 using the 1-click upgrade v4.13.0.
I attached a screen record
https://watch.screencastify.com/v/4fcqMPLVodx83dTMgbjt





> Hi @khouloudbelguith, in my case that was helpful. After logging into incognito mode tinymce starts working properly.
> 
> Thanks

@przemokon, Great :slightly_smiling_face: 

Thank you all!

**Comment by gs2k84** [comment]
After resetting the cache directly at your hosting provider, incognito mode works. It still needs to fix the operation on the standard version of the browser tab.

**Comment by khouloudbelguith** [comment]
Thank you for your feedback.
As it is a cache browser.
I will close the issue, feel free to create a new issue when needed.

Thanks!
